src/types.ts

```diff
@@ -795,6 +795,7 @@ export type SearchOptions = {
 export type StreamChatOptions = AxiosRequestConfig & {
   browser?: boolean;
   logger?: Logger;
+  serverConnectUser?: boolean;
```

Comment:
Not sure but this name sounds confusing and could make understanding its purpose harder.

Reply:
Any suggestions?

Reply:
naming is hard but a verbose guess to make it harder to misinterpret, `enableBecomingOnlineFromServerSide`

Reply:
What if we drop the option and force developer to stop using server key? If there's a use case for this, you can always use two different clients :)